src/plugins/intel_gpu/src/graph/impls/ocl/resample.cpp

```diff
@@ -150,8 +150,16 @@ struct resample_impl : typed_primitive_impl_ocl<resample> {
         us_params.pads_begin = convert_pads(primitive->pads_begin, dimsNum);
         us_params.pads_end = convert_pads(primitive->pads_end, dimsNum);
 
-        for (size_t i = 0; i < primitive->scales.size(); i++) {
-            us_params.axesAndScales[convert_axis(primitive->axes[i], dimsNum)] = primitive->scales[i];
+        auto scales = primitive->scales;
+        bool scales_calc_mod = primitive->shape_calc_mode == resample::InterpolateOp::ShapeCalcMode::SCALES;
```

Comment:
This condition checks whether `shape_calc_mode` is scales mode. but this commit only sets for size mode as follows: 

`InterpolateOp::ShapeCalcMode shape_calc_mode = InterpolateOp::ShapeCalcMode::SIZES`

When the `shape_calc_mode` will be set to scale mode?

Reply:
shape_calculation_mode is set when the primitive constructor is called in CreateInterpolateOp() and contains the value from ngraph::op::v4::Interpolate op